hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsNetworkStatistics.java

```diff
@@ -223,13 +223,7 @@ public void testAbfsHttpResponseStatistics() throws IOException {
        *
        * bytes_received - This should be equal to bytes sent earlier.
        */
-      long extraCalls = 0;
```

Comment:
getResponsesBeforeTest will be the total number of calls before the open and read operations. Basically the removed part was buggy because here we do the calculation wrongly by including the extracalls which is already covered in the getResponsesBeforeTest.